@xen-orchestra/lite/src/components/infra/InfraHostItem.vue

```diff
@@ -14,6 +14,7 @@
     >
       {{ host.name_label || "(Host)" }}
       <template #actions>
+        <InfraAction v-if="isPoolMaster" :icon="faStar" class="master-icon" />
```

Comment:
I'm unfamiliar with the code, but I have some functional questions:
- is there an [`alt` attribute](https://devdocs.io/html/attributes#alt-attribute) for the icon?
- is there a tooltip or another kind of info in case the user does not know what this icon mean?

Reply:
- It's a svg tag
-  I added a tooltip for it.